daprdocs/content/en/reference/cli/dapr-run.md

```diff
@@ -23,6 +23,7 @@ dapr run [flags] [command]
 
 | Name                           | Environment Variable | Default                                                                            | Description                                                                                          |
 | ------------------------------ | -------------------- | ---------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------- |
+| `--kubernetes`, `-k`             |            |                                                                                    | Running Dapr on Kubernetes, and used for [Multi-App Run template files on Kubernetes]({{< ref multi-app-dapr-run >}}).                                                            |
```

Comment:
I would put this middle to last in the table, not the first entry.
Also can we add an example into the examples section how to use the multi-app run here with the Kubernetes option